src/main/java/org/folio/consortia/domain/mapper/TenantMapper.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.consortia.domain.mapper;
+
+import org.folio.consortia.domain.dto.Tenant;
+import org.folio.consortia.domain.entity.TenantEntity;
+import org.mapstruct.Mapper;
+import org.mapstruct.NullValueCheckStrategy;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring", nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS)
+public interface TenantMapper {
```

Comment:
I personally like just using standard spring converter for these cases, here is example: https://github.com/folio-org/mod-notes/blob/ef49bde812030b67c99630d93fefbc694b1c8df7/src/main/java/org/folio/notes/domain/converter/StringToOrderDirectionConverter.java